script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_8.txt

```diff
@@ -91,7 +91,7 @@ In that case, I will be calling you again in an hour."
 "An hour from now, you mean for dinner?"
 "Yes. Until then, please make yourself at home."
 <Page27>
-Hisui still says that without an expression on her face.
+Hisui still says that with a blank expression on her face.
```

Comment:
This formulation makes it sound like she has been saying it for a long time ("she still says that"), perhaps reorder to 

> Hisui says this with a blank expression still on her face

orig text:

> 翡翠はやっぱり無表情で言ってくる。

Reply:
on second thought
> Hisui says even this with a blank expression.